extensions/components/com_redcore/admin/views/translation/tmpl/modal-edit_fields.php

```diff
@@ -0,0 +1,192 @@
+<?php
```

Comment:
this file is identical/very similar as redcore admin edit form... 
1. You should put that code into a layout file
2. Load both from admin and here from that layout file